docs/messaging-service/index.md

```diff
@@ -0,0 +1,74 @@
+# Messaging Service
+
+## What is the Messaging service?
+
+The *Messaging service* is functionality within the *OneWelcome Identity Cloud* that enables you to configure and send notifications to your customers via SMS
+or email. The Messaging service serves as a notification gateway for other OneWelcome products to unify the configuration of the notification channels.
+
+The Messaging service can send out SMS and emails with the data directly embedded in the request, or can use templates from [*Customer
+Self-Styling*](../self-styling/index.md). Using templates gives you the option to adjust the look of the notifications.
+
+Expected structure in Customer Self-Styling is:
+
+```
+tenant-specific-folder:
+    | 
+   messaging-service:
+        |
+        email:
+        |   |
+        |   html:
+        |       - template.mustache
+        |       - template2.mustache
+        |   text:
+        |       - template.mustache
+        |       - template2.mustache
+        sms:
+        |   - template.mustache
+        messages:
+            - messages.properties
+            - messages_en.properties
+            - messages_pl.properties
+            - messages_otherlocalecode.properties              
+```
+
+The Messaging service uses the [Mustache template language](https://github.com/samskivert/jmustache) for templating. Templates can be localized and sent in a
+language specified in the request. All messages files MUST contain all translation keys. The Messaging service does not merge the default file with localized
```

Comment:
unnecessary line break

Reply:
Single line break in markdown does not make a line break in the rendered text. Its just for convenience in our editor to not have very long lines of text that you need to scroll horizontally. You need 2 line breaks to have a real line break in markdown. See
![Screenshot 2021-12-06 at 08 46 45](https://user-images.githubusercontent.com/55274735/144807281-2de11a58-9901-4ec7-8b38-2fb4afc62718.png)